gcp/service-account-key/mainf.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
+  required_version  = ">=0.12.3"
+}
+
+provider "google" {
+  project = var.project
```

Comment:
Please pin a version to the provider - there are major changes between v2 and v3.
And add an optional `region` and `zone` configuration as a variable with defaults.

Reply:
Regarding the `region` and `zone` - this is a global resources so there's no need to specify them.
Added version - 82f5396